syncd/syncd_flex_counter.h

```diff
@@ -190,6 +221,12 @@ class FlexCounter
         void saiUpdateSupportedBufferPoolCounters(sai_object_id_t bufferPoolId,
                                                   const std::vector<sai_buffer_pool_stat_t> &counterIds,
                                                   sai_stats_mode_t statsMode = SAI_STATS_MODE_READ_AND_CLEAR);
+
+        std::vector<sai_port_stat_t> saiCheckSupportedPortDebugCounters(sai_object_id_t portId,
+                                                                        const std::vector<sai_port_stat_t> &counterIds);
```

Comment:
missing _In_ tag on each param, also dont put param in the first line, but on next when function defined